valohai_cli/commands/execution/run/dynamic_run_command.py

```diff
@@ -121,12 +122,17 @@ def convert_param_to_option(self, parameter: Parameter) -> Option:
         Convert a Parameter into a click Option.
         """
         assert isinstance(parameter, Parameter)
+        help = parameter.description
+        is_multiple = parameter.multiple is not None
+        if is_multiple:
+            help = "(Multiple) " + (help if help else "")
```

Comment:
```suggestion
        help = (parameter.description or "")
        is_multiple = parameter.multiple is not None
        if is_multiple:
            help = f"(Multiple) {help}"
```
might be a bit more succinct...